DcCore/DcCore/DC/Wrapper.swift

```diff
@@ -2,6 +2,86 @@ import Foundation
 import UIKit
 import AVFoundation
 
+public class DcAccounts {
+
+    /// The application group identifier defines a group of apps or extensions that have access to a shared container.
+    /// The ID is created in the apple developer portal and can be changed there.
+    let applicationGroupIdentifier = "group.chat.delta.ios"
+    var accountsPointer: OpaquePointer?
+
+    public init() {
+        var version = ""
+        if let appVersion = Bundle.main.infoDictionary?["CFBundleShortVersionString"] as? String {
+            version += " " + appVersion
+        }
+
+        if let sharedDbLocation = FileManager.default.containerURL(forSecurityApplicationGroupIdentifier: applicationGroupIdentifier) {
+            accountsPointer = dc_accounts_new("iOS\(version)", sharedDbLocation.path)
```

Comment:
the directory we're passing here is not empty, there may be the old messenger.db and messenger.db-blobs in.

to avoid problems, i would use a subdirectory as "accounts". core will create the subdirectory if it does not exist (as well as all the needed subdirectories for the accounts).